spec/models/habit_builder_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+describe HabitBuilder do
+  describe "#build" do
+    let(:user) { FactoryGirl.create(:user) }
+    it "associates current user properly" do
+      params = { name: 'brushing teeth', frequency: { "daily" => "1" } }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.